Automattic/pocket-casts-ios
# Playback 2024: Top Spot

| 📘 Part of: #2250 |
|:---:|

Completes #2316 

| | |
|--|--|
| <video src="https://github.com/user-attachments/assets/fbd86513-cca7-4ebc-a049-12aefcbd3166"> | <video src="https://github.com/user-attachments/assets/e27a4962-a258-455f-8217-947483c025a1"> |

## To test

1. Enable the endOfYear2024 feature flag & restart the app
2. Navigate to the Profile and tap Playback 2024 Prompt
3. Navigate through the stories until you hit this Top Podcast screen
4. Verify layout and animations work properly

## Checklist
